modules/tracktion_engine/playback/tracktion_DeviceManager.cpp

```diff
@@ -1122,10 +1122,13 @@ void DeviceManager::audioDeviceIOCallbackInternal (const float** inputChannelDat
                     juce::FloatVectorOperations::clear (dest, numSamples);
 
             currentCpuUsage = std::min (0.9, currentCpuUsage * 0.99);
+
+            numCpuOverloads++;
+            std::cout << "Audio Engine: CPU OVERLOAD!\n";
```

Comment:
nit: Let's remove this one or at least use their logger. I feel like it might be a bit loud at times.